src/main/java/edu/wpi/first/shuffleboard/widget/NetworkTableTreeWidget.java

```diff
@@ -38,7 +39,7 @@
    * Compares tree items alphabetically.
    */
   private static final Comparator<TreeItem<NetworkTableEntry>> alphabetical
-      = Comparator.comparing(item -> item.getValue().getKey().toLowerCase());
+      = Comparator.comparing(item -> item.getValue().getKey().toLowerCase(Locale.getDefault()));
```

Comment:
Using Locale.getDefault() here is exactly identical to existing behavior. Why make it explicit?